Source/WTF/wtf/MathExtras.h

```diff
@@ -145,9 +124,7 @@ inline double roundTowardsPositiveInfinity(double value) { return std::floor(val
 inline float roundTowardsPositiveInfinity(float value) { return std::floor(value + 0.5f); }
 
 // std::numeric_limits<T>::min() returns the smallest positive value for floating point types
-template<typename T> constexpr T defaultMinimumForClamp() { return std::numeric_limits<T>::min(); }
-template<> constexpr float defaultMinimumForClamp() { return -std::numeric_limits<float>::max(); }
-template<> constexpr double defaultMinimumForClamp() { return -std::numeric_limits<double>::max(); }
+template<typename T> constexpr T defaultMinimumForClamp() { return std::numeric_limits<T>::lowest(); }
```

Comment:
This is now out of sync with the comment above it. 

The commit message does not explain why this change is being made. Please add that explanation.